app/models/application_record.rb

```diff
@@ -92,7 +92,13 @@ def move_to_dashboard
 
   def self.move_to_dashboard
     records = where('date_for_later <= ?', Time.zone.today)
-    records.each(&:move_to_dashboard)
+    records.each do |record|
+      record.move_to_dashboard
+      record.position.subscriptions.each do |sub|
```

Comment:
Hmm, seems like this could be handled in the instance `move_to_dashboard` method (and then this could be put back the way it was).